rsmtool/utils.py

```diff
@@ -626,9 +648,8 @@ def get_output_directory_extension(directory, experiment_id):
         and are in the list of possible output extensions.
     """
     extension = 'csv'
-    extensions_identified = [os.path.splitext(name)[-1].replace('.', '')
-                             for name in os.listdir(directory)
-                             if any(name.endswith(ext) for ext in POSSIBLE_EXTENSIONS)]
+    extensions_identified = [ext for ext in POSSIBLE_EXTENSIONS
+                             if has_files_with_extension(directory, ext)]
 
     extensions = set(extensions_identified)
```

Comment:
Actually, if you need a set, just use a set comprehension (`{ext for ext in ...}`) instead of a list comprehension above.

Reply:
Good call. :)

I'll make that change and fix a couple of the minor things Anastassia mentioned.